src/main/anovos/data_transformer/transformers.py

```diff
@@ -1731,9 +1740,39 @@ def imputation_sklearn(
         a few handful of them. (Default value = [])
     method_type
         "KNN", "regression".
-        "KNN" option trains a sklearn.impute.KNNImputer. "regression" option trains a sklearn.impute.IterativeImputer (Default value = "KNN")
+        "KNN" option trains a sklearn.impute.KNNImputer. "regression" option trains a sklearn.impute.IterativeImputer
+        (Default value = "regression")
+    use_sampling
+        Boolean argument - True or False. This argument is used to determine whether to use random sample method on
+        source and target dataset, True will enable the use of sample method, otherwise False.
+        It is recommended to set this as True for large datasets. (Default value = True)
+    sample_method
+        If use_sampling is True, this argument is used to determine the sampling method.
+        "stratified" for Stratified sampling, "random" for Random Sampling.
+        For more details, please refer to https://docs.anovos.ai/api/data_ingest/data_sampling.html.
+        (Default value = "random")
+    strata_cols
+        If use_sampling is True and sample_method is "stratified", this argument is used to determine the list
+        of columns used to be treated as strata. For more details, please refer to
+        https://docs.anovos.ai/api/data_ingest/data_sampling.html. (Default value = "all")
+    stratified_type
+        If use_sampling is True and sample_method is "stratified", this argument is used to determine the stratified
+        sampling method. "population" stands for Proportionate Stratified Sampling,
+        "balanced" stands for Optimum Stratified Sampling. For more details, please refer to
+        https://docs.anovos.ai/api/data_ingest/data_sampling.html. (Default value = "population")
     sample_size
-        Maximum rows for training the sklearn imputer (Default value = 500000)
+        If use_sampling is True, this argument is used to determine maximum rows for training the sklearn imputer
+        (Default value = 10000)
+    sample_seed
+        If use_sampling is True, this argument is used to determine the seed of sampling method.
+        (Default value = 42)
+    persist
+        Boolean argument - True or False. This argument is used to determine whether to persist on
+        binning result of source and target dataset, True will enable the use of persist, otherwise False.
+        It is recommended to set this as True for large datasets. (Default value = True)
+    persist_option
+        If persist is True, this argument is used to determine the type of persist.
+        (Default value = pyspark.StorageLevel.MEMORY_AND_DISK)
```

Comment:
I have added the list of the pyspark.StorageLevel from pyspark official documentation in here.
For the label, according to Killian comment on #268 ,https://github.com/anovos/anovos/pull/268#discussion_r948473258, it would be better to just set it to pyspark.StorageLevel.MEMORY_AND_DISK to avoid confusion.

Reply:
okay noted.